packages/lodestar/src/tasks/tasks/interopSubnetsJoiningTask.ts

```diff
@@ -25,9 +25,12 @@ export class InteropSubnetsJoiningTask implements ITask {
 
   public async run(): Promise<void> {
     const forkDigest = this.chain.currentForkDigest;
+    const attNets: AttestationSubnets = this.config.types.AttestationSubnets.defaultValue();
     for (let i = 0; i < this.config.params.RANDOM_SUBNETS_PER_VALIDATOR; i++) {
-      this.subscribeToRandomSubnet(forkDigest);
+      const subnet = this.subscribeToRandomSubnet(forkDigest);
+      attNets[subnet] = true;
     }
+    this.network.metadata.attnets = attNets;
```

Comment:
I wonder if it would be better to do this in single place like in:
- this.network.gossip.subscribeToAttestationSubnet
- this.network.gossip.unsubscribeFromAttestationSubnet

Reply:
yeah I did that